### PR TITLE
Fix build instructions msg

### PR DESCRIPTION
**Pull request by danmosemsft**
Address https://github.com/dotnet/corefx/issues/15955

@willdean

### CODE REVIEW

**Comment by willdean** [comment]
Definitely better, thanks.   Looks like there is a comment in the code above about VS2013+ which is maybe wrong?

File: src/Native/build-native.cmd [hunk 1]
```diff
@@ -50,9 +50,9 @@ if "%VisualStudioVersion%"=="14.0" (
 )
 
 :MissingVersion
-:: Can't find VS 2013+
-echo Error: Visual Studio 2015 required
-echo        Please see https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/developer-guide.md for build instructions.
+:: Can't find VS 2015
+echo Error: Visual Studio 2015 required
```
Comment (mellinoe) [inline src/Native/build-native.cmd]: nit: Extra spaces

Comment (danmosemsft) [inline src/Native/build-native.cmd]: I think I"ll pass on that one this time

Comment (weshaggard) [inline src/Native/build-native.cmd]: FWIW I think we should actually fall back to using the .NET Core msbuild if we don't detect msbuild on the path. We already have everything we need in the Tools directory to do that.

**Pull request merged**